crates/bevy_input/src/mouse.rs

```diff
@@ -126,6 +126,40 @@ pub struct MouseWheel {
     pub y: f32,
 }
 
+/// Touchpad magnification event with two-finger pinch gesture.
+///
+/// Positive delta values indicate magnification (zooming in) and
+/// negative delta values indicate shrinking (zooming out).
+///
+/// ## Platform-specific
+///
+/// - Only available on **`macOS`**.
+#[derive(Event, Debug, Clone, Copy, PartialEq, Reflect, FromReflect)]
+#[reflect(Debug, PartialEq)]
```

Comment:
Could you also add `FromReflect` to this attribute?

Reply:
Isn't that already the case? It is the last argument to `#[derive(...)]`.

Reply:
Sorry, to clarify I’m referring to adding a `ReflectFromReflect` registration like `#[reflect(Debug, PartialEq, FromReflect)]`